runtime/core/portable_type/c10/c10/macros/Macros.h

```diff
@@ -396,6 +396,8 @@ extern SYCL_EXTERNAL void __assert_fail(
     const char* file,
     unsigned int line,
     const char* func);
+#elif (defined(__EMSCRIPTEN__))
```

Comment:
ah we mirror pytorch/pytorch for anything under c10/.

Can you submit this change upstream. Then we will have to pin bump and youll have it